src/App.js

```diff
@@ -10,30 +10,37 @@ class App extends Component {
         super();
 
         this.state = {
-            data: null
+            inputData: [],
```

Comment:
InputData is an example of a bad naming. It's actually not inputData, may be something like "tasksList"